### ISSUE TITLE
What is the loss printed out in the de-en demo?

### ISSUE THREAD

**Issue by anglil** [tensorflow/tensor2tensor#117]
Is it per word perplexity or averaged cross entropy (neg log likelihood) or something else?

**Comment by lukaszkaiser** [comment]
The loss is neg-log-likelihood rescaled with label smoothing together with weight-decay. It's hard to interpret, that's why we have eval metrics that show the true neg-log-perplexity (no label smoothing or weight decay).
